ibis/backends/bigquery/__init__.py

```diff
@@ -460,6 +460,7 @@ def create_table(
             sql_select = self.compile(table)
             table_ref = f"`{project_id}`.`{dataset}`.`{name}`"
             self.raw_sql(f'CREATE TABLE {table_ref} AS ({sql_select})')
+        return self.table(name, database=database)
```

Comment:
codecov says this isn't covered, but this is unrelated to this change. Open an issue to add a test for create_table() on bigquery?

Reply:
It's reporting no coverage because we don't run the bigquery tests until after merge, because we don't want PRs to have access to creds.

Reply:
No need for an issue right now.